schemas/telemetry/new-profile/new-profile.1.schema.json

```diff
@@ -0,0 +1,777 @@
+                    }
+                  }
+                },
+                "binary": {
+                  "type": "object",
+                  "properties": {
+                    "model": {
+                      "type": [
+                        "string",
+                        "null"
+                      ]
+                    },
+                    "revision": {
+                      "type": [
+                        "string",
+                        "null"
+                      ]
+                    }
+                  }
+                },
+                "system": {
+                  "type": "object",
+                  "properties": {
+                    "model": {
+                      "type": [
+                        "string",
+                        "null"
+                      ]
+                    },
+                    "revision": {
+                      "type": [
+                        "string",
+                        "null"
+                      ]
+                    }
+                  }
+                }
+              }
+            },
+            "isWow64": {
+              "type": "boolean"
+            },
+            "memoryMB": {
+              "type": "number"
+            },
+            "os": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "version": {
+                  "type": ["string", "integer"]
+                },
+                "kernelVersion": {
+                  "type": "string"
+                },
+                "servicePackMajor": {
+                  "type": "number"
+                },
+                "servicePackMinor": {
+                  "type": "number"
+                },
+                "windowsBuildNumber": {
+                  "type": "number"
+                },
+                "windowsUBR": {
+                  "type": ["number", "null"]
+                },
+                "installYear": {
+                  "type": "number"
+                },
+                "locale": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      },
+      "required": [
+        "build",
+        "partner",
+        "settings",
+        "system"
+      ]
+    },
+    "id": {
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "meta": {
+      "type": "object"
+    },
+    "payload": {
+      "type": "object",
+      "properties": {
+        "reason": {
+          "type": "string",
+          "enum": [
+            "shutdown",
+            "startup"
+          ]
+        }
+      },
+      "required": [
+        "reason"
+      ]
+    },
+    "type": {
+      "type": "string",
+      "enum": [
+        "new-profile"
+      ]
+    },
+    "version": {
+      "type": "number",
+      "minimum": 4,
```

Comment:
Woops, I was a bit too quick approving there - looks like the filename should be `new-profile.4.schema.json` rather than `new-profile.1.schema.json`